bin/vault

```diff
@@ -138,6 +138,7 @@ main() {
     fi
 
     if [[ "${PASSWORD:-}" == "" ]]; then
+        (>&2 printf "${RED}No VAULT_KEY found\n$NC")
```

Comment:
There are use cases where dops is used without VAULT_KEY. It should be better to fail if no VAULT_KEY is defined only if there is at least 1 file to decrypt.

Reply:
Moving the vault_key definition test in set_password and call it when encrypting/decrypting is a good idea.